In this study, we reconstructed a GEM for
the reference strain 477 and extended the GEM construction workflow for six different strains of H.
influenzae, two from each of the three main phylogenetic clades. All of the simulations and model
validations were performed using cobratoolbox of the MATLAB. Based on the strain specific study, H.
influenzae core genome consists of 1805 reactions and 373 genes and pangenome consists of 2121
reactions and 576 genes. All of the metabolic models were analysed with respect to auxotrophy for
amino acids, vitamins and carbon sources. Vitamin B3 and vitamin B5 were found to be auxotrophic for
all the strains, while arginine and histidine were auxotrophic amino acids. GEMs were used to grow
biomass and polysaccharide capsule under experimental conditions provided in the literature and they
were verified with respect to growth rate and growth trends. Gene essentiality analysis identified the
essential genes required for bacterial growth and can thus be focused for drug targets.

![image](https://github.com/TejasMKale/GEM_reconstruction/assets/110218791/c5349b59-8428-4aec-b36b-231c74547c70)

